src/marqo/tensor_search/tensor_search.py

```diff
@@ -267,9 +267,53 @@ def _autofill_index_settings(index_settings: dict):
 
 
 def get_stats(config: Config, index_name: str):
-    doc_count = HttpRequests(config).post(path=F"{index_name}/_count")["count"]
+    """Returns the number of documents and vectors in the index.
+
+    The _count API counts top-level documents.
```

Comment:
I suggest moving these details out of method docstring and into a standard comment. The docstring is generally a blackbox description of what the method does and details that are relevant for usage of the method (e.g., side effects, time complexity, special/edge case considerations etc)